akka-docs/rst/java/code/docs/http/javadsl/server/directives/FileUploadDirectivesExamplesTest.java

```diff
@@ -14,121 +14,132 @@
 import akka.util.ByteString;
 import org.junit.Ignore;
 import org.junit.Test;
+import scala.concurrent.duration.Duration;
+import scala.concurrent.duration.FiniteDuration;
+
 import java.io.File;
 import java.nio.charset.Charset;
 import java.nio.file.Files;
 import java.util.Arrays;
 import java.util.HashMap;
 import java.util.Map;
 import java.util.concurrent.CompletionStage;
+import java.util.concurrent.TimeUnit;
 import java.util.function.BiFunction;
 
 public class FileUploadDirectivesExamplesTest extends JUnitRouteTest {
 
-    @Test
-    public void testUploadedFile() {
-        //#uploadedFile
-        // function (FileInfo, File) => Route to process the file metadata and file itself
-        BiFunction<FileInfo, File, Route> infoFileRoute =
-                (info, file) -> {
-                    // do something with the file and file metadata ...
-                    file.delete();
-                    return complete(StatusCodes.OK);
-                };
-
-
-        final Route route = uploadedFile("csv", infoFileRoute);
-
-        Map<String, String> filenameMapping = new HashMap<>();
-        filenameMapping.put("filename", "data.csv");
-
-        akka.http.javadsl.model.Multipart.FormData multipartForm =
-                Multiparts.createStrictFormDataFromParts(Multiparts.createFormDataBodyPartStrict("csv",
-                        HttpEntities.create(ContentTypes.TEXT_PLAIN_UTF8,
-                        "1,5,7\n11,13,17"), filenameMapping));
-
-        // test:
-        testRoute(route).run(HttpRequest.POST("/").withEntity(
-                multipartForm.toEntity(HttpCharsets.UTF_8, BodyPartRenderer
-                .randomBoundaryWithDefaults())))
-                .assertStatusCode(StatusCodes.OK);
-        //#
+  @Override
+  public FiniteDuration awaitDuration() {
+    return FiniteDuration.create(3, TimeUnit.SECONDS); // because file access
```

Comment:
well, 3 seconds is the recommended value unless you have a good reason for shorter. The default of `500.millis` is too short in my opinion. Changing the default would be good, but I don't know if there are assumptions in existing tests.